modules/ROOT/pages/dialog-configuration.adoc

```diff
@@ -45,7 +45,7 @@ tinymce.activeEditor.windowManager.open({
 |title |string |required |The title of the dialog. This will display in the header of the dialog.
 |body |`+panel+` or `+tabpanel+` component |required |The specification for the xref:dialog-components.adoc[body component].
 |buttons |`+FooterButton[]+` |required |An array of xref:dialog-footer-buttons.adoc[footer buttons] to render in the footer of the dialog.
-|size |`+'normal'+`, `+'medium'+` or `+'large'+` |optional |default: `+normal+` - Dialog size options.
+|size |`+'normal'+`, `+'medium'+` or `+'large'+` |optional |default: `+normal+`. `+large+` sets the dialog to a large fixed height. `+medium+` sets the dialog to render with only the height required to fits its components. `+normal+` *[does what?]*. `inline` dialogs do not support `size: 'large'`.
```

Comment:
```suggestion
|size |`+'normal'+`, `+'medium'+` or `+'large'+` |optional |default: `+normal+`. `+large+` sets the dialog to a large fixed height. `+medium+` sets the dialog to render with only the height required to fits its components. 
```
Don't think ``+normal+` *[does what?]*. `inline` dialogs do not support `size: 'large'`.`  should be in the doc

Reply:
It has been [edited](https://github.com/tinymce/tinymce-docs/commit/f17935b3e66bcde403b4fab215bcdd6bb9a3d285) out.